dwpicker/selection.py

```diff
@@ -15,33 +15,26 @@ def select_targets(shapes, selection_mode='replace'):
     hovered = [s for s in shapes if s.hovered]
     targets = [t for s in hovered for t in s.targets() if cmds.objExists(t)]
 
-    if selection_mode in ('add', 'replace', 'invert'):
-        try:
-            return cmds.select(list(targets), add=selection_mode == 'add')
-        except ValueError:
-            raise NameclashError(targets)
-    elif selection_mode == 'remove':
-        selection = [n for n in cmds.ls(sl=True) if n not in targets]
-        try:
-            return cmds.select(selection)
-        except ValueError:
-            raise NameclashError(targets)
-
-    # Invert selection
```

Comment:
I've removed the code bellow as it was dead code and cannot be reached as `selection_mode` can only be `add`, `replace`, `invert` or `remove`.